instrumentation/trilogy/test/opentelemetry/instrumentation/trilogy/instrumentation_test.rb

```diff
@@ -346,6 +346,46 @@
       end
     end
 
+    describe 'when propagator is set to none' do
+      let(:config) { { propagator: :none } }
+
+      it 'does not inject context' do
+        sql = +'SELECT * from users where users.id = 1 and users.email = "test@test.com"'
+        original_sql = sql.dup
+        expect do
+          client.query(sql)
+        end.must_raise Trilogy::Error
+        _(sql).must_equal original_sql
+      end
+    end
+
+    describe 'when propagator is set to nil' do
+      let(:config) { { propagator: nil } }
+
+      it 'does not inject context' do
+        sql = +'SELECT * from users where users.id = 1 and users.email = "test@test.com"'
+        original_sql = sql.dup
+        expect do
+          client.query(sql)
+        end.must_raise Trilogy::Error
+        _(sql).must_equal original_sql
+      end
+    end
+
+    describe 'when propagator is set to vitess' do
+      let(:config) { { propagator: :vitess } }
```

Comment:
Validating that it's a string but passing in a symbol
```suggestion
      let(:config) { { propagator: 'vitess' } }
```